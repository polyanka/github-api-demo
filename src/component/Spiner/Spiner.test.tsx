import { shallow } from 'enzyme';
import * as React from 'react';

import { Spiner } from './index';

describe('<Spiner />', () => {
  it('renders correctly Spiner component', () => {
    const wrapper = shallow(<Spiner />);
    expect(wrapper).toMatchSnapshot();
  });
});
