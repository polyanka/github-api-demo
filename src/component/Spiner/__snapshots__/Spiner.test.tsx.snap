// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Spiner /> renders correctly Spiner component 1`] = `
<WithStyles(ForwardRef(Grid))
  alignItems="center"
  container={true}
  direction="column"
  justify="center"
  spacing={4}
  style={
    Object {
      "height": 260,
    }
  }
>
  <WithStyles(ForwardRef(Grid))
    item={true}
    xs={2}
  >
    <WithStyles(ForwardRef(CircularProgress)) />
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Grid))>
`;
